### PR TITLE
Add draggable scrollable sheet

### PR DESCRIPTION
**Pull request by sanju-developer**

### CODE REVIEW

File: lib/screens/signup/signup_with_phone/signup_with_phone.dart [hunk 1]
```diff
@@ -21,6 +22,14 @@ class _SignupWithPhoneState extends State<SignupWithPhone> {
     });
   }
 
+  bool validateIndianPhone(String value) {
```
Comment (shubhamtanwar23) [inline lib/screens/signup/signup_with_phone/signup_with_phone.dart]: isValidIndianPhone is a much better name

File: lib/screens/signup/signup_with_phone/signup_with_phone.dart [hunk 2]
```diff
@@ -122,6 +131,9 @@ class _SignupWithPhoneState extends State<SignupWithPhone> {
                                       } else if (value.length < 10 &&
                                           isSignUpBtnClicked) {
                                         return 'Please enter valid phone no!';
+                                      } else if (validateIndianPhone(value) &&
```
Comment (shubhamtanwar23) [inline lib/screens/signup/signup_with_phone/signup_with_phone.dart]: I think you forgot not

File: lib/screens/signup/signup_with_phone/signup_with_phone.dart [hunk 3]
```diff
@@ -13,6 +13,7 @@ class _SignupWithPhoneState extends State<SignupWithPhone> {
   final _formKey = GlobalKey<FormState>();
   final phoneController = TextEditingController();
   bool isSignUpBtnClicked = false;
+  String phoneRegex = r'^[789]\d{9}$$';
```
Comment (shubhamtanwar23) [inline lib/screens/signup/signup_with_phone/signup_with_phone.dart]: make it final

File: lib/screens/signup/signup_with_phone/signup_with_phone.dart (hunk 1 above)
Comment (sanju-developer) [reply lib/screens/signup/signup_with_phone/signup_with_phone.dart]: updated

File: lib/screens/signup/signup_with_phone/signup_with_phone.dart (hunk 2 above)
Comment (sanju-developer) [reply lib/screens/signup/signup_with_phone/signup_with_phone.dart]: updated

File: lib/screens/signup/signup_with_phone/signup_with_phone.dart (hunk 3 above)
Comment (sanju-developer) [reply lib/screens/signup/signup_with_phone/signup_with_phone.dart]: updated

**Comment by sanju-developer** [comment]
@shubhamtanwar23 ,please check PR comments are updated. If it's fine then please merge

**Pull request merged**